src/Documentation/grains/index.md

```diff
@@ -65,10 +65,49 @@ public class PlayerGrain : Grain, IPlayerGrain
 }
 ```
 
+Grains can have two interfaces and one implementation.
```

Comment:
This is an odd statement, especially considering that on line 17 we already said kind of the same, but in a more correct way